### ISSUE TITLE
ConfigFileFinder is removed from flake8

### ISSUE THREAD

**Issue by wyuenho** [tylerwince/flake8-bandit#33]
https://github.com/tylerwince/flake8-bandit/blob/4043bc51fcb2327da4b0ae82cab3b7a4a8bdbe87/flake8_bandit.py#L10

This flake8 [commit](https://github.com/PyCQA/flake8/commit/65c893728ec7834f95afdd06736107b4f3a5ea7a) removed it.

There are a few things from flake8's head that I need, but I'd also like to use flake8-bandit, so this is a bit of a pickle for me

**Comment by staticdev** [comment]
Now flake8-bandit does not work anymore with latest release from flake8 (5.0.0). It will throw:

```
ImportError: cannot import name 'ConfigFileFinder' from 'flake8.options.config'
```

**Comment by wyuenho** [comment]
I believe bandit is best used by itself now since it has its own configuration lookup algorithm. It's either this or fix this plugin to use bandit's configuration lookup routine instead.

**Comment by lsorber** [comment]
Created PR #35 to attempt to address this issue.

**Comment by scop** [comment]
Thanks. I see some 4.x versions have been tagged here, but PyPI is still at 3.0.0. Any ETA for an update over there?

**Comment by staticdev** [comment]
I am now successfully using bandit directly in my projects.. recommend the same. Eg. https://github.com/staticdev/human-readable/pull/412

**Comment by skwashd** [comment]
@scop once #39 is merged and v4.1.1 is tagged PyPI should get the new release automagically.

**Comment by fredrikaverpil** [comment]
@skwashd doesn't seem like there was an automagic pypi release. It's still [at 3.0.0 there](https://pypi.org/project/flake8-bandit/#history), released in March.

**Comment by skwashd** [comment]
@scop the CI bug was fixed and 4.1.1 was pushed to PyPI earlier today.
